scripts/source_code.py

```diff
@@ -1,8 +1,14 @@
+from _ast import AST
```

Comment:
what is the difference between module `ast` and `_ast`?

Reply:
_ast is a low-level implementation where defined the type 'AST'. I just checked ast internally imports the _ast module. So never mind, I have removed this code.